src/lib/components/inbox/conversations/__tests__/index-tests.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react"
+import "react-native"
+import * as renderer from "react-test-renderer"
+
+import Inbox from "../"
+
+it("looks correct when rendered", () => {
```

Comment:
had to rename it to `index-tests` because the implementation file is `index.ts` and danger checks for a matching test file